lua/packer.lua

```diff
@@ -123,6 +127,9 @@ packer.init = function(user_config)
 end
 
 packer.make_commands = function()
+  vim.cmd [[command! -nargs=+ -complete=customlist,v:lua.require'packer.snapshot'.completion.create PackerSnapshot  lua require('packer').snapshot(<f-args>)]]
+  vim.cmd [[command! -nargs=+ -complete=customlist,v:lua.require'packer.snapshot'.completion.rollback PackerRollback  lua require('packer').rollback(<f-args>)]]
```

Comment:
These should probably be called `PackerSnapshotRollback` and `PackerSnapshotDelete` especially the delete on It can be easily mistaken for something else like plugin uninstallation or deletion on packer_compiled file.